firefox-ios/Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -501,4 +515,11 @@ extension TabScrollingController: UIScrollViewDelegate {
         }
         return true
     }
+
+    func setOffset(y: CGFloat, for scrollView: UIScrollView) {
```

Comment:
this method can be private, also might be better to just put it in the `TabScrollController` since that's the only place its being used.